install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -54,38 +54,42 @@ data:
         restartPolicy: Always
       containers:
       - name: istio-proxy
-        image: {{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}
+        image: {{ "[[ if (isset .ObjectMeta.Annotations \"sidecar.istio.io/proxyImageOverride\") -]]" }}
```

Comment:
How about 'sidecar.istio.io/proxyImage' ? All options override the default, no need to be in the name,

@rshriram  @ZackButcher @nmittler @louiscryan  Any other suggestions for the name ? Will likely be used post 0.8, to allow users to specify custom builds of envoy (extra plugins, different base images, etc).